AISKU/src/InternalConstants.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Any strings added to the "InternalConstants.ts" are automatically used to scan for and replace any possible references